### DISCUSSION TITLE
i cant run ( npm run dev)

### DISCUSSION THREAD

**Discussion by mustafaxpp** [laravel/framework#36233] [Q&A]
I cant run npm run dev and I can't solve this problem I
![Capture](https://user-images.githubusercontent.com/59034013/107728138-e930aa00-6cf5-11eb-889e-a318a251cf95.PNG)
 wish someone helps me

**Comment by browner12** [comment]
Github discussions is not the place to ask for help.

Please close this issue and try on a forum like StackOverflow.

**Reply by mustafaxpp** [reply]
okay, How I can close this issue ?

**Reply by browner12** [reply]
https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository#deleting-a-discussion

**Reply by mustafaxpp** [reply]
that's how to delete discussions in your repository and that's not my repository so it's not the correct link
